op-batcher/batcher/sync_actions.go

```diff
@@ -69,14 +68,8 @@ func computeSyncActions[T channelStatuser](
 		"syncStatus.unsafeL2", newSyncStatus.UnsafeL2.TerminalString(),
 	)
 
-	safeL2 := newSyncStatus.SafeL2
-	if preferLocalSafeL2 {
-		// This is preferred when running interop, but not yet enabled by default.
-		safeL2 = newSyncStatus.LocalSafeL2
-	}
```

Comment:
I suggest we keep the definition as `safeL2 := newSyncStatus.LocalSafeL2` because it's referenced in so many places. Also reduces the diff.